tests/index.js

```diff
@@ -697,4 +697,35 @@ describe('rule', () => {
       return validate(data, rules).then(check).catch(check)
     })
   })
+
+  /*
+   * Regular Expression
+   * ----------------------------------------------------- */
+  describe('#regular_expression', () => {
```

Comment:
Tests looks good. For future referece, you can check https://github.com/laravel/framework/blob/5.4/tests/Validation/ValidationValidatorTest.php#L2264